pkg/git.go

```diff
@@ -183,12 +190,12 @@ func (p *GitPackage) Install(ctx context.Context, name, dir, version string) (st
 	destPath := path.Join(dir, name)
 
 	pkgh := sha256.Sum256([]byte(fmt.Sprintf("jsonnetpkg-%s-%s", strings.Replace(name, "/", "-", -1), strings.Replace(version, "/", "-", -1))))
-	// using 16 bytes should be a good middle ground between length and collision resistance
-	tmpDir, err := ioutil.TempDir(filepath.Join(dir, ".tmp"), hex.EncodeToString(pkgh[:16]))
-	if err != nil {
-		return "", errors.Wrap(err, "failed to create tmp dir")
+
+	// Create the cache directory for the current package
+	cacheDir := filepath.Join(dir, "cache", hex.EncodeToString(pkgh[:16]))
```

Comment:
Maybe this should be:
a. A default "well-known" location (for instance `~/.cache/jsonnet-bundler` ?)
b. overridable with an environment variable ?
-> that could simply be to follow the XDG_CACHE_DIR scheme ?